aws/rust-runtime/aws-runtime/src/auth.rs

```diff
@@ -270,8 +469,8 @@ pub mod sigv4 {
                 "test",
             );
             let operation_config = SigV4OperationSigningConfig {
-                region: SigningRegion::from_static("test"),
-                service: SigningService::from_static("test"),
+                region: Some(SigningRegion::from_static("test")),
+                service: Some(SigningService::from_static("test")),
```

Comment:
I needed them to be optional so that the endpoint auth scheme config could set their value when they weren't otherwise set (where previously, it would panic because they didn't have values).